operators/src/util/raster_stream_to_geotiff.rs

```diff
@@ -132,22 +136,34 @@ fn gdal_writer<T: Pixel + GdalType>(
     let mut options = vec![
         RasterCreationOption {
             key: "COMPRESS",
-            value: "LZW",
+            value: COMPRESSION_FORMAT,
         },
         RasterCreationOption {
             key: "TILED",
             value: "YES",
         },
+        RasterCreationOption {
+            key: "ZLEVEL",
```

Comment:
should be "LEVEL" https://gdal.org/drivers/raster/cog.html?highlight=level

Reply:
It is level for COG but ZLEVEL for GeoTiff